share/functions/help.fish

```diff
@@ -86,6 +86,12 @@ function help --description 'Show help for the fish shell'
                 set fish_browser $cmd[1]
             end
 
+            # Try and see if powershell.exe is an option to avoid UNC path error
+            # https://ss64.com/nt/cmd.html Launching CMD/batch files from a UNC path
+            if type -q powershell.exe
+                set fish_browser powershell.exe
```

Comment:
This should be *inside* the WSL if above.

Make it 

```fish
and set -l cmd (command -s powershell.exe /mnt/c/Windows/System32/powershell.exe cmd.exe /mnt/c/Windows/System32/cmd.exe)
```

Reply:
updated conditional with the above `and` statement and set `fish_browser` as `$cmd[1]`